lib/hexpm/accounts/users.ex

```diff
@@ -452,4 +452,12 @@ defmodule Hexpm.Accounts.Users do
        valid?: false
      }}
   end
+
+  def has_role(user,role) do
+    if user != nil do
+      if user.role == role do true end
```

Comment:
==/2 already returns true|false so we can write simply:

```suggestion
      user.role == role
```